Resources/views/base.html.twig

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
```

Comment:
I don't think this file is necessary, plz take a look at other integrated bundles